hack/ci/install-kind.sh

```diff
@@ -17,19 +17,21 @@
 #     along with Nexus Operator.  If not, see <https://www.gnu.org/licenses/>.
 
 set -e
-default_kind_version="v0.8.1"
+default_kind_version=v0.8.1
 
 if [[ -z ${KIND_VERSION+x} ]]; then
     KIND_VERSION=$default_kind_version
     echo "Using default KIND version ${KIND_VERSION}"
 fi
+GOPATH=$(go env GOPATH)
 
-if [ ! -f ./bin/kind ]; then
-    echo "Installing KIND"
-    curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-amd64
-    chmod +x ${PWD}/kind &&
-        mkdir -p ./bin &&
-        mv ${PWD}/kind ./bin/kind
+if [[ $(which kind) ]]; then
+  echo "---> kind is already installed. Please make sure it is the required ${KIND_VERSION} version before proceeding"
 else
```

Comment:
`which` should be avoided when possible. It spaws a different process and implementation may vary from one distribution to another, so it may not even set a return value which is used in the comparation here. In this case we can stick to the `command` built-in:
```suggestion
if ! command -v kind &> /dev/null; then
  echo "---> kind is already installed. Please make sure it is the required ${KIND_VERSION} version before proceeding"
else
```
More details [here](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script).

Reply:
Updated to use command, didn't use & at the end, it is working as expected without it